### PR TITLE
VIR-5-fix-styling

### PR DESCRIPTION
**Pull request by eimi-codes**
I've added more styling information based upon the setting up guide.

### CODE REVIEW

**Comment by eimi-codes** [comment]
@KatrinaAS I've added more commits, can you re-review please? TIA

File: Gemfile [hunk 1]
```diff
@@ -17,8 +17,14 @@ gem "minima", "~> 2.5"
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.12"
   gem "jekyll-remote-theme","~> 0.4.3"
+  gem "jekyll-include-cache"
+  gem "jekyll-paginate"
+  gem "jekyll-sitemap"
+  gem "jekyll-gist"
```
Comment (GaeaKat) [inline Gemfile:23]: Need to find out what each of these do, and either use them, or remove them in the long run I think

File: _config.yml [hunk 2]
```diff
@@ -28,10 +28,34 @@ github_username: ViridianVille
...
 plugins:
   - jekyll-remote-theme
+  - jekyll-paginate
+  - jekyll-sitemap
+  - jekyll-gist
   - jekyll-feed
+  - jekyll-include-cache
+
+
+minimal_mistakes_skin: "mint"
```
Comment (GaeaKat) [inline _config.yml:42]: I do like mint theme!

**Review by GaeaKat — APPROVED** [review]
Great PR @aimeesunflower

**Pull request merged**